modules/appeals_api/spec/support/rswag_config.rb

```diff
@@ -993,6 +998,11 @@ def decision_reviews_sc_response_schemas
                     type: 'string',
                     format: 'date-time',
                     example: '2018-07-30T17:31:15.958Z'
+                  },
+                  finalStatus: {
+                    description: 'Indicates whether the status of the submission is final. Submissions with a final_status of true will no longer update to a new status.',
```

Comment:
The fields in this API are camel-case rather than snake-case, so `final_status` should instead be `finalStatus`.